data/migrations/versions/e8ed3fb547da_alter_manifestblob_id.py

```diff
@@ -0,0 +1,37 @@
+"""alter manifestblob id
+
+Revision ID: e8ed3fb547da
+Revises: 3634f2df3c5b
+Create Date: 2025-05-13 09:45:39.152681
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "e8ed3fb547da"
+down_revision = "3634f2df3c5b"
+
+import sqlalchemy as sa
+
+
+def upgrade(op, tables, tester):
+    bind = op.get_bind()
+    if bind.engine.name == "postgresql":
+        op.execute(
+            """
+            ALTER TABLE manifestblob ALTER COLUMN id TYPE BIGINT;
+        """
+        )
+        op.execute(
+            """
+            ALTER SEQUENCE manifestblob_id_seq AS BIGINT;
+        """
+        )
+        op.execute(
+            """
+            ALTER SEQUENCE manifestblob_id_seq MAXVALUE 9223372036854775807;
+        """
+        )
+
+
+def downgrade(op, tables, tester):
+    pass
```

Comment:
can you add the downgrade steps. It's basically the reverse BIGINT -> INT

Reply:
You can’t safely migrate from BIGINT to INT without checking the data first. Any existing values outside the INT range will cause truncation or errors.